cartographer_ros/src/node_options.h

```diff
@@ -23,25 +23,22 @@
 #include "cartographer/common/port.h"
 #include "cartographer/mapping/map_builder.h"
 
+#include "sensor_bridge.h"
+
 namespace cartographer_ros {
 
 // Top-level options of Cartographer's ROS integration.
 struct NodeOptions {
   ::cartographer::mapping::proto::MapBuilderOptions map_builder_options;
+  SensorBridgeOptions sensor_bridge_options;
```

Comment:
Update the documentation by prefixing `sensor_bridge.` in the appropriate places?